history.txt

```diff
@@ -1,6 +1,27 @@
 OMERO version history
 =====================
 
+5.3.2 (May 2017)
+----------------
+
+This is a bug-fix.
+
+Improvements include:
+
+- improve populate_metadata plugin
+- fix deletion of a range of objects from CLI
+- allow to add textual annotation without namespace at import from CLI
```

Comment:
textual annotations without a namespace can now be added at import using the CLI